modules/phase_field/src/kernels/KKSACBulkC.C

```diff
@@ -29,6 +29,18 @@ KKSACBulkC::KKSACBulkC(const InputParameters & parameters) :
     _prop_d2Fadca2(getMaterialPropertyDerivative<Real>("fa_name", _ca_name, _ca_name)),
     _prop_d2Fbdcb2(getMaterialPropertyDerivative<Real>("fb_name", _cb_name, _cb_name))
 {
+  //Resize to number of coupled variables (_nvar from KKSACBulkBase constructor)
+  _off_diag_a.resize(_nvar);
+
+  // Iterate over all coupled variables
+  for (unsigned int i = 0; i < _nvar; ++i)
+  {
+    MooseVariable *cvar = _coupled_moose_vars[i];
+
+    // get second partial derivatives wrt ca and other coupled variable
+    _off_diag_a[i] = &getMaterialPropertyDerivative<Real>("fa_name", _ca_name, cvar->name());
+  }
+
```

Comment:
Yeah... there are a few places where you left an empty line before a closing `}`. If you could remove those that'd be great.